### DISCUSSION TITLE
How to track multiple trainings and models using single DVC repo and yaml?

### DISCUSSION THREAD

**Discussion by aveek-brahmachari** [treeverse/dvc#6949] [Help]
A git project may have many different models for different purposes. It does not seem to be a solution to have multiple DVC repos and track each training/experiment/model separately.

So the question is if there are multiple tasks: Task A, Task B, Task C, Task D,......Task Z and there are various models trained for the same, how can we use DVC to efficiently do this ? Did not find any example explaining this.

**Comment by daavoo** [comment]
You can initialize multiple DVC Repos in separated subdirectories under the same git project:

https://dvc.org/doc/command-reference/init#initializing-dvc-in-subdirectories

**Reply by aveek-brahmachari** [reply]
Is there any more elegant method of doing this ? If I have 100 models that I train in a project, it would mean I would need 100 DVC repos to track them too.

**Reply by karajan1001** [reply]
You can create [for each stages](https://dvc.org/doc/user-guide/project-structure/pipelines-files#foreach-stages) which could create batches of similar stages.
